db/seeds.rb

```diff
@@ -3,7 +3,7 @@
 
 User.delete_all
 
-user = User.create(email: "email@email.com", password: "password")
+user = User.create(email: "email@email.com", password: "****",password_confirmation: "****" )
```

Comment:
Line is too long. [93/80]<br>Space missing after comma.<br>Space inside parentheses detected.